lib/dbl.php

```diff
@@ -129,28 +138,25 @@ function sanitized_dsn() {
 
     /** @return ?\mysqli */
     function connect() {
-        assert(($this->name ?? "") !== "");
-        if ($this->socket) {
-            $dblink = new mysqli($this->host, $this->user, $this->password, "", $this->port, $this->socket);
-        } else if ($this->port !== null) {
-            $dblink = new mysqli($this->host, $this->user, $this->password, "", $this->port);
-        } else {
-            $dblink = new mysqli($this->host, $this->user, $this->password);
-        }
+	assert(($this->name ?? "") !== "");
+
+        $dblink = new mysqli();
+        $dblink->ssl_set($this->sslKey, $this->sslCert, $this->sslCaCert, null, null);
+        $dblink->options(MYSQLI_OPT_SSL_VERIFY_SERVER_CERT, $this->sslVerifyServerCert ?? false);
```

Comment:
Seems like this option should default to true

Reply:
Tweaked. Makes sense if there is an extra "dbSsl" option.